### DISCUSSION TITLE
i am using .NET 7.0 but

### DISCUSSION THREAD

**Discussion by theerudito** [Megabit/Blazorise#5005] [General]
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1605	Warning As Error: Detected package downgrade: Microsoft.Extensions.Logging.Abstractions from 7.0.1 to 7.0.0. Reference the package directly from the project to select a different version. 
 Bootstrap -> Blazorise.Bootstrap5 1.3.1 -> Microsoft.Extensions.Logging.Abstractions (>= 7.0.1) 
 Bootstrap -> Microsoft.Extensions.Logging.Abstractions (>= 7.0.0)	Bootstrap	C:\Users\ewine\source\repos\MAUI-APP\Bootstrap\Bootstrap.csproj	1

**Comment by stsrki** [comment]
I believe this is a duplicate of #4835. As far as I am aware, the problem is in the Xamarin packages, which have an inconsistent version.

**Comment by theerudito** [comment]
It is not xamarin, it is MAUI Hybrid with the Blazor version of the SDK 7.0 but in the SDK 6.0 if you install the Blazorice package
